renderer/components/Request/Request.js

```diff
@@ -30,6 +31,7 @@ class Request extends React.Component {
     invoice: PropTypes.object,
     isAnimating: PropTypes.bool,
     isProcessing: PropTypes.bool,
+    maxOneTimeReceive: PropTypes.number.isRequired,
```

Comment:
Since this prop has been marked as required we're going to get warnings in the storybook for not setting it. Consider making non-required. Alternatively ensure that we set it everywhere where `Receive` is used (storybook).

Reply:
Actually, this is not an issue since we use the connected container in the storybook and not the raw component.